t/shipto-exceptions.t

```diff
@@ -0,0 +1,110 @@
+# for the most part it is copy-paste from billto-exceptions.t
+# but not totally, as some tests are specific to shipping
+
+use strict;
+use warnings;
+use Test::More;
+use Test::Fatal qw(exception dies_ok);
+
+use Module::Runtime 'use_module';
+
+my $shipto_c = use_module('Business::CyberSource::RequestPart::ShipTo');
+
```

Comment:
I'm missing the coverage context. It is either about sandbox being able to swallow all the fields, or about the integrity of the ShipTo object. 

- If former, I agree, and add all possible fields. It is, though, related to xt/ tests, not to t/shipto-exceptions.t

- If latter, t/shipto-exceptions.t seem to cover all the fail cases. t/shipto.t must be extended with missing fields.